packages/v2/contracts/ANTv2.sol

```diff
@@ -42,26 +48,16 @@ contract ANTv2 is IERC20 {
         _;
     }
 
-    constructor(uint256 chainId, address initialMinter) public {
-        DOMAIN_SEPARATOR = keccak256(
-            abi.encode(
-                keccak256('EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)'),
-                keccak256(bytes(name)),
-                keccak256(bytes('1')),
-                chainId,
-                address(this)
-            )
-        );
-
+    constructor(address initialMinter) public {
         _changeMinter(initialMinter);
     }
 
-    function _validateSignedData(address signer, bytes32 encodedData, uint8 v, bytes32 r, bytes32 s) internal view {
+    function _validateSignedData(address signer, bytes32 encodeData, uint8 v, bytes32 r, bytes32 s) internal view {
```

Comment:
@facuspagnuolo Coming back to an earlier conversation, I ended up picking `encodeData` because it's the nomenclature used by ERC712 :).